man/kernel-install.xml

```diff
@@ -64,16 +76,19 @@
     below.</para>
 
     <para><command>kernel-install</command> will run the executable files ("plugins") located in the
-    directory <filename>/usr/lib/kernel/install.d/</filename> and the local administration directory
-    <filename>/etc/kernel/install.d/</filename>.  All files are collectively sorted and executed in lexical
-    order, regardless of the directory in which they live. However, files with identical filenames replace
-    each other.  Files in <filename>/etc/kernel/install.d/</filename> take precedence over files with the
-    same name in <filename>/usr/lib/kernel/install.d/</filename>. This can be used to override a
-    system-supplied executables with a local file if needed; a symbolic link in
-    <filename>/etc/kernel/install.d/</filename> with the same name as an executable in
-    <filename>/usr/lib/kernel/install.d/</filename>, pointing to <filename>/dev/null</filename>, disables the
-    executable entirely. Executables must have the extension <literal>.install</literal>; other extensions
-    are ignored.</para>
+    <filename>install.d</filename> directories listed in the SYNOPIS. Per the usual rules, the directory
+    under <filename>/usr/lib/</filename> should be used by packages, the directory under
+    <filename>/etc/</filename> can be used for local configuration, and the directory under
+    <filename>/run/</filename> can be used for temporary overrides. Note that
+    <filename>/run/kernel/install.d/</filename> has the highest priority. All files are collectively sorted
```

Comment:
For consistency, even if there are no generators here, I think it's better to keep the same priority as everything else. We really want the usr + etc + run configuration handling to be the same everywhere, so that people don't have to think about what tool they are using and what special cases come with it. If the same approach just works everywhere in the same way, you don't have to think about it, you just know how it works because it's the same pattern.

Reply:
So I guess the motivation for this changed order is the usecase described in the commit msg?

I must say, I am not convinced about the altered order here either. I appears to me that we should support .d/ dropins specifically for `/run/kernel/install.conf`, and that's how the usecase should be covered. I mean, as I understand that's all you are interested in?overriding the generator listed in there and some other fields?

If you want to mask some plugins then the regular /run/ ordering should be fine, since plugins generally live in /usr/, not /etc/